src/mirco_contactstatus_kokkos.h

```diff
@@ -0,0 +1,39 @@
+#ifndef SRC_CONTACTSTATUS_KOKKOS_H_
+#define SRC_CONTACTSTATUS_KOKKOS_H_
+
+#include "mirco_kokkostypes_kokkos.h"
+
+namespace MIRCO
+{
+  /**
+   * @brief The aim of this function is to compute to nodes in contact in the current iteration
+   *
+   * @param[out] xvf x-coordinates of the points in contact in the previous iteration.
+   * @param[out] yvf y-coordinates of the points in contact in the previous iteration.
+   * @param[out] pf Contact force at (xvf,yvf) predicted in the previous iteration.
+   * @param[out] nf Number of nodes in contact in the previous iteration
+   * @param[in] y Solution containing force
+   * @param[in] xv0 x-coordinates of the points in contact in the previous iteration.
+   * @param[in] yv0 y-coordinates of the points in contact in the previous iteration.
```

Comment:
Documentation of these two parameter is the same as for the first two. Which one is correnct? Which one needs to be updated?

Reply:
`xv0` and `yv0` are the coordinates of the points predicted to be in contact in the current iteration.